app/controllers/runs_controller.rb

```diff
@@ -1,7 +1,5 @@
 require 'uri'
 
-require 'speedrundotcom'
```

Comment:
Not sure if this was no longer needed because of the new autoloader, or because at some point `/lib` was added to the autoloading path.